File: 1002.java
import java.io.*;

public class Main {
    public static void main(String[] s) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        double[] x = new double[6];
        double D, R, r;

        for (int T = Integer.parseInt(br.readLine()); T>0; T--) {
            s = br.readLine().split(" ");
            for (int i=0; i<6; i++) x[i] = Double.parseDouble(s[i]);
            D = Math.sqrt((x[0]-x[3])*(x[0]-x[3]) + (x[1]-x[4])*(x[1]-x[4]));
            R = Math.max(x[2], x[5]); r = Math.min(x[2], x[5]);

            if (D == 0) if (R == r) sb.append(-1); else sb.append(0);
            else if (D == R + r || D == R - r) sb.append(1);
            else if (D > R + r) sb.append(0);
            else if (D > R - r) sb.append(2);
            else sb.append(0);
            sb.append('\n');
        }

        System.out.print(sb);
    }
}

File: 1002.py
import sys

T = int(sys.stdin.readline())
for _ in range(T):
    x1, y1, r1, x2, y2, r2 = map(int, sys.stdin.readline().split())
    a,b,c = 0,0,0
    if r2 > r1:
        a,b,c = x1, y1, r1
        x1, y1, r1 = x2, y2, r2
        x2, y2, r2 = a, b, c
    d = ((x1-x2)**2 + (y1-y2)**2)
    if x1 == x2 and y1 == y2 and r1 == r2:
        print(-1)
    elif r1**2 >= d: # 내부
        if d < (r1-r2)**2:
            print(0)
        elif d == (r1-r2)**2:
            print(1)
        else:
            print(2)
    elif r1**2 < d: # 외부
        if d > (r1 +r2)**2:
            print(0)
        elif d == (r1 +r2)**2:
            print(1)
        else:
            print(2)
